packages/common/src/relay/calling/Call.ts

```diff
@@ -62,24 +62,36 @@ export default class Call implements ICall {
     return Boolean(this.id)
   }
 
-  get type(): string {
+  get type() {
     const { type } = this.options.device
     return type
   }
 
   get from(): string {
-    const { params: { from_number = '' } = {} } = this.options.device
-    return from_number
+    if (this.device.type === 'phone') {
+      return this.device?.params?.from_number ?? ''
+    } else if (this.device.type === 'sip') {
+      return this.device?.params?.from ?? ''
+    }
+    // @ts-ignore
+    return this.device?.params?.from ?? ''
   }
 
   get to(): string {
-    const { params: { to_number = '' } = {} } = this.options.device
-    return to_number
+    if (this.device.type === 'phone') {
+      return this.device?.params?.to_number ?? ''
+    } else if (this.device.type === 'sip') {
+      return this.device?.params?.to ?? ''
+    }
+    // @ts-ignore
+    return this.device?.params?.to ?? ''
   }
 
   get timeout(): number {
-    const { params: { timeout = DEFAULT_CALL_TIMEOUT } = {} } = this.options.device
-    return timeout
+    if (this.device.type === 'phone') {
+      return this.device?.params?.timeout ?? DEFAULT_CALL_TIMEOUT
+    }
+    return null
```

Comment:
Can we add a check to see if its present for `sip`, if it is, use it, otherwise return null? We'll be adding it soon, might as well make it more backwards compatible.

Reply:
Sure! Done in 30dc5f11c7de46098d435ce94aa8541407470044